modules/plugins/assistant/avante/avante-nvim.nix

```
@@ -76,7 +76,7 @@ in {
         };
 
         dual_boost = {
-          enabled = mkEnableOption "dual_boost mode.";
+          enabled = mkEnableOption "Dual_boost mode.";
```

```suggestion
          enabled = mkEnableOption "`dual_boost` mode.";
```

I think `dual_boost` is a technical term and must be made into inline code, without the capitalization.